mysql-test/include/percona_log_encrypt_change.inc

```diff
@@ -0,0 +1,28 @@
+# InnoDB transparent tablespace data encryption
+# This test case will verify that log encryption type can't be changed
+
+call mtr.add_suppression("Redo log encryption mode can't be switched without stopping the server and recreating the redo logs");
+
+--source include/have_innodb.inc
+--source include/not_embedded.inc
+
+# Test: changing when it is explicitly set
+
+SELECT @@innodb_redo_log_encrypt;
+--eval SET GLOBAL innodb_redo_log_encrypt=$LOG_ENCRYPT_OTHER_TYPE
+SELECT @@innodb_redo_log_encrypt;
+
+# Test: changing when it is turned off, but the logs ARE encrypted
+
+--eval SET GLOBAL innodb_redo_log_encrypt=OFF
+--eval SET GLOBAL innodb_redo_log_encrypt=$LOG_ENCRYPT_OTHER_TYPE
+SELECT @@innodb_redo_log_encrypt;
+
+--let SEARCH_FILE= $MYSQLTEST_VARDIR/log/mysqld.1.err
```

Comment:
Error log grep antipattern here, the error log contents will have all the previous test content, please look into assert_only_after=CURRENT_TEST: test_name of assert_grep.inc, if not practical, then use a custom error log path